File: CORREDOR/Tipos Constructores/Java/Carro.java
package Java;

public class Carro extends Transporte implements Vehiculo{
    String marca;
    int modelo;

    //Constructor sin parámetros
    public Carro() {
        this.tipo = "Carro";
        this.marca = "Toyota";   
        this.modelo = 2020;
    }

    @Override
    public void mover() {
        System.out.println("El carro está en movimiento.");
    }

    @Override
    void informacion() {
        System.out.println("Tipo de transporte: " + tipo);
        System.out.println("Marca: " + marca);
        System.out.println("Modelo: " + modelo);
    }
}

File: CORREDOR/Tipos Constructores/C#/MiProyecto/Carro.cs
public class Carro : Transporte, Vehiculo{
    public string Marca { get; set; }
    public int Modelo { get; set; }
    //Constructor Sin Parametros
    public Carro(){
        Tipo = "Carro";
        Marca = "Toyota";
        Modelo = 2020;
    }

    public void Mover(){
        Console.WriteLine("El carro está en movimiento.");
    }

    public override void Informacion(){
        Console.WriteLine($"Tipo de transporte: {Tipo}");
        Console.WriteLine($"Marca: {Marca}");
        Console.WriteLine($"Modelo: {Modelo}");
    }
}
